get_trace_data.py

```diff
@@ -22,16 +24,27 @@
 
     parsed_args = arg_parser.parse_args()
 
+    # Create dirs
+    bbox = parsed_args.bbox
+    output_dir, output_tmp_dir = util.initialize_dirs(bbox)
+
```

Comment:
Do this in main script so we can point the map matching process to `output_dir` later.